### ISSUE TITLE
Pretrained Inception V1 model not predicting proper labels

### ISSUE THREAD

**Issue by blake-varden** [tensorflow/models#1358]
Please go to Stack Overflow for help and support: http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug or a feature request.
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:
models/slim/nets/
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 14.04
- **TensorFlow installed from (source or binary)**:
binary
- **TensorFlow version (use command below)**:
1.1.0-rc1
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
CUDA 8/cuDNN 5.1
- **GPU model and memory**:
Titan X Pascal
- **Exact command to reproduce**:

```
import tensorflow as tf
from tensorflow.contrib.slim.nets import inception
import numpy as np
slim = tf.contrib.slim

num_classes = 1001
tf_image = tf.placeholder(tf.float32, [None, 224,224,3])
tf_labels = tf.placeholder(tf.int32, [None, num_classes])
with slim.arg_scope(inception.inception_v1_arg_scope()):
    logits, end_points = inception.inception_v1(tf_image,
                                                num_classes=num_classes,
                                                is_training=False)
predictions = end_points['Predictions']


variables_to_restore = slim.get_variables_to_restore()
# path to my downloaded models
init_assign_fn = slim.assign_from_checkpoint_fn('/nas/weights/inception_v1.ckpt',
                                                variables_to_restore)

session=tf.Session()
init_assign_fn(session)
# http://www.4x4expedition.se/Image/expedition/baboon.jpg
import cv2
image = cv2.imread('../baboon.jpg')
image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)

with session.as_default():
    pred = session.run(end_points['Predictions'], feed_dict={tf_image:[image]})
# prints 916
print np.argmax(pred)    

# https://s-media-cache-ak0.pinimg.com/736x/96/3d/56/963d562a02e0a4a49633b1e1026071fb.jpg
image = cv2.imread('../husky.jpg')
image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)

with session.as_default():
    pred = session.run(end_points['Predictions'], feed_dict={tf_image:[image]})
# prints 916    
print np.argmax(pred)    
```
You can collect some of this information using our environment capture script: https://github.com/tensorflow/tensorflow/tree/master/tools

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem

Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

I instantiate the inception v1 model with the pretrained weights and pass a baboon image through and a husky image through (both 224x224).  The argmax of the predictions for both images is 916.  It is also unclear with the inception models what label->human label mapping should be used.  Regardless it doesnt appear as though the pretrained weights are successfully working with the net to create an accurate prediction.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by drpngx** [comment]
@nealwu did we update the inception models?

**Comment by nealwu** [comment]
It looks like the checkpoints are from Aug/Sept 2016, so there's a small possibility that we've made some changes that could have broken the checkpoints. However I would first suggest posting on StackOverflow with your code and making sure that your code doesn't have any errors first. If it is indeed a problem with the checkpoints, feel free to reopen and let us know.

As far as the labels, it looks like 916 corresponds to "yurt," so that's definitely not what you're looking for. I did find this online that might help you with mapping model labels to human labels: https://gist.github.com/yrevar/942d3a0ac09ec9e5eb3a. My labels are offset by 1 from this file though, since I have a 0 category for "background" (followed by 1 for "tench, Tinca tinca"). In any case, none of the labels near 916 are what you're looking for.

**Comment by LucasMahieu** [comment]
Hi, I have exactly the same problem than @blake-varden. I also have "yurt" instead of "husky" or other dogs...

Any idea to solve this issue ?
It seems that the training was made with a different order of the human readable list ...
